crates/uv-virtualenv/src/bare.rs

```diff
@@ -322,6 +353,8 @@ impl WindowsExecutable {
         match self {
             WindowsExecutable::Python => "venvlauncher.exe",
             WindowsExecutable::Pythonw => "venvwlauncher.exe",
+            WindowsExecutable::PyPy => "venvlauncher.exe",
+            WindowsExecutable::PyPyw => "venvwlauncher.exe",
```

Comment:
I have no idea whether this is correct or not.

Reading a comment below I seemed to understand that the shims changed name in Python 3.13 but after installing it doesn't seem like it.

Also I think this shouldn't apply to PyPy for the time being since it supports up to Python 3.10 only.

Reply:
Can you add a comment explaining this?